core/src/main/kotlin/com/malinskiy/marathon/io/FileManager.kt

```diff
@@ -29,6 +29,12 @@ class FileManager(private val output: File) {
         return File(resultsFolder, filename)
     }
 
+    fun createFile(fileType: FileType, pool: DevicePoolId, device: DeviceInfo, fileName: String): File {
```

Comment:
We try to not have any fileName-dependent methods here. In this case the filename is also a constant `marathon_junit_report`. Would you please change this another method that doesn't have any file name?

Reply:
Hi, I am bit confused here. But as I understand, I've removed the createFile with fileName and handled the file creation within junitWriter itself.